content/04.results.md

```diff
@@ -56,3 +56,40 @@ Points are colored by `short_histology` of the samples they represent.
 B) Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `short_histology`.
 C) Heatmap of xCell fraction values for scores and cell types with high variance, with samples ordered by `short_histology`.
 ](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/transcriptomic-overview.png){#fig:transcriptomic-overview}
+
+### Quantify Telomerase Activity using Gene Expression Data
+
+We used gene expression data sets from both poly-A and stranded RNA-Seq protocols to predict telomerase activities using EXTEND as shown in Supplementary Table {@tab:telomerase-activities}A , Table {@tab:telomerase-activities}B, Table {@tab:telomerase-activities}C and Table {@tab:telomerase-activities}D.
+
+![Telomerase activities for pediatric brain tumors.
+A) Telomerase Scores for stranded FPKM data.
+B) Telomerase Scores for poly-A FPKM data.
+C) Telomerase Scores for stranded Counts data.
+D) Telomerase Scores for poly-A Counts data.
+](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/telomerase-activity-prediction/results)
+
+For both platforms, EXTEND scores were highly consistent between FPKM versus counts data as shown in Figure {@fig:comparison-plots}A and Figure {@fig:comparison-plots}B.
+  
+![Scatter plots comparing telomerase activities via Spearman correlation for counts with FPKM format.
+A) poly-A platform .
+B) stranded platform.
+](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/telomerase-activity-prediction/plots/PTBA_GE_Score_AllScatter.pdf){@fig:comparison-plots}
+
+We next examined telomerase activity across different brain tumor histological subtypes using data from stranded FPKM protocol. 
+Across the short histological groups, we found germinomas had the highest EXTEND scores. 
+However, teratoma, another type of germ cell tumor, had much lower scores as shown in Figure {@fig:distribution-plots}A.
+Teratomas were previously reported to have negligible or low activities [@doi:10.1093/jnci/91.15.1321]. 
+Benign neoplasia including dysplasia and hemangioblastomas had the lowest scores. 
+Among common pediatric brain tumors, medulloblastomas had higher scores than high grade astrocytic tumors, ependymomas and low-grade astrocytoma. 
+Across broad groups, benign tumors and precancerous lesions were among those with the lowest scores whereas tumors of pineal region and embryonal tumors have the highest scores as shown in Figure {@fig:distribution-plots}B. 
```

Comment:
+ Across broad **histology** groups
+ Add comma before "whereas"
+ For consistency, I would phrase this as "pineal region and embryonal tumors"

As a general recommendation, it sometimes helps to just reference the figure in parentheses rather than adding the extra phrase "as shown in". Removing some of those words can help with the flow of ideas.